js/join.js

```diff
@@ -350,8 +351,9 @@ submit.addEventListener('click', async () => {
     .catch((err) => {
       alert(`Error in saving user data ${err}`);
     });
+  window.localStorage.setItem('hasJoinVisited', false);
```

Comment:
When do we set and reset hasJoinVisited ?

Reply:
if user complete the join section

Reply:
for more info see [this](https://github.com/Real-Dev-Squad/website-www/pull/425#discussion_r1131811425)